core/src/main/java/org/keycloak/representations/account/ConsentScopeRepresentation.java

```diff
@@ -23,15 +23,22 @@ public class ConsentScopeRepresentation {
 
     private String name;
 
+    /**
+     * @deprecated Use {@link #displayText} instead. This field will be removed in KC 27.0.
+     */
+    @Deprecated
     private String displayTest;
```

Comment:
Could you please remove this deprecated field, and only keep the new one? Then the old getter/setter would only point to that new field. IMHO this should be enough to create the right JSON, and parse the JSON correctly, assuming people will send one or the other.